src/Lean/Environment.lean

```diff
@@ -88,11 +88,6 @@ structure EnvironmentHeader where
   -/
   trustLevel   : UInt32       := 0
```

Comment:
We should move `trustLevel` to `Kernel.Environment`. It is not being used since we moved from Lean 3, but we will eventually be used again.